test/unit/middleware/files.spec.js

```diff
@@ -148,6 +148,85 @@ describe("static server with trailing slash customization", () => {
       .end(done);
   });
 
+  describe("i18n", () => {
+    beforeEach(() => {
+      fs.outputFileSync(".tmp/intl/es/index.html", "hola", "utf8");
```

Comment:
How about a test that it defaults to ALL_ca if the file exists there, but doesn't exist in fr_ca?

Reply:
Yyyyes. I think I can!